### ISSUE TITLE
Update code style documentation

### ISSUE THREAD

**Issue by anyzelman** [Algebraic-Programming/ALP#169]
This is to update some very-out-of-date documents regarding code style that still reside in both develop and master; it is not intended as an issue that would close #5. This is a **blocker** to v0.7. It will be handled after #156 .

**Comment by anyzelman** [comment]
Addressing changes are merged into develop (#172 ), closing issue
